kolibri/plugins/coach/assets/src/views/lessons/LessonSummaryPage/UpdatedResourceSelection.vue

```diff
@@ -0,0 +1,310 @@
+<template>
+
+  <div class="select-resource">
+    <div>
+      <ResourceSelectionBreadcrumbs
+        v-if="topic"
+        :ancestors="[...topic.ancestors, topic]"
+        :channelsLink="channelsLink"
+        :topicsLink="topicsLink"
+      />
+
+      <ContentCardList
+        :contentList="contentList"
+        :showSelectAll="canSelectAll"
+        :viewMoreButtonState="viewMoreButtonState"
+        :selectAllChecked="selectAllChecked"
+        :selectAllIndeterminate="selectAllIndeterminate"
+        :contentIsChecked="contentIsChecked"
+        :contentIsIndeterminate="contentIsIndeterminate"
+        :contentHasCheckbox="showCheckbox"
+        :contentCheckboxDisabled="contentCheckboxDisabled"
+        :contentCardLink="contentLink"
+        :showRadioButtons="!multi"
+        @changeselectall="handleSelectAll"
+        @change_content_card="toggleSelected"
+        @moreresults="fetchMoreResources"
+      />
+    </div>
+  </div>
+
+</template>
+
+
+<script>
+
+  import { computed } from '@vue/composition-api';
+  import commonCoreStrings from 'kolibri/uiText/commonCoreStrings';
+  import { ContentNodeKinds } from 'kolibri/constants';
+  import ContentCardList from '../../lessons/LessonResourceSelectionPage/ContentCardList.vue';
+  import ResourceSelectionBreadcrumbs from '../../lessons/LessonResourceSelectionPage/SearchTools/ResourceSelectionBreadcrumbs.vue';
+  import { injectResourceSelection } from './sidePanels/LessonResourceSelection/useResourceSelection';
+  import {
+    ResourceContentSource,
+    ResourceSelectionView,
+  } from './sidePanels/LessonResourceSelection/constants';
+
+  export default {
+    name: 'UpdatedResourceSelection',
+    components: {
+      ContentCardList,
+      ResourceSelectionBreadcrumbs,
+    },
+    mixins: [commonCoreStrings],
+    setup(props) {
```

Comment:
This is a really clean component overall - I really appreciate the simplicity in how the composable modules are implemented <3